src/Interface/IsCommutativeMonoid.agda

```diff
@@ -0,0 +1,26 @@
+{-# OPTIONS --safe #-}
+
+module Interface.IsCommutativeMonoid where
+
+open import Algebra
+open import Prelude
+
+record IsCommutativeMonoid' c ℓ Carrier : Set (sucˡ (c ⊔ˡ ℓ)) where
+  infixl 7 _∙_
+  infix  4 _≈_
+  field
+    _≈_                 : Carrier → Carrier → Set ℓ
+    _∙_                 : Carrier → Carrier → Carrier
+    ε                   : Carrier
```

Comment:
Shouldn't these operations come from superclasses, i.e. 
```agda
record IsCommutativeMonoid' c ℓ Carrier : Set (sucˡ (c ⊔ˡ ℓ)) where
  infixl 7 _∙_
  infix  4 _≈_
  field
    ⦃ setoid ⦄ : Setoid Carrier
    ⦃ semigroup ⦄ : Semigroup Carrier
    ⦃ monoid ⦄ : Monoid Carrier
    isCommutativeMonoid : IsCommutativeMonoid {c} _≈_ _◇_ ε
```
(I think we don't yet have `Setoid` so at least the other two for now)

Reply:
I like @omelkonian's suggestion.  Maybe less obvious/straightforward to understand/use/apply (at least for novices), but it seems like the "right" way to do things from an abstraction point of view.